gtfs/spec/en/reference.md

```diff
@@ -32,6 +33,25 @@ This section defines terms that are used throughout this document.
 * **Field optional** - The field column may be omitted from your feed. If you choose to include an optional column, each record in your feed must have a value for that column. You may include an empty string as a value for records that do not have values for the column. Some optional fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string.
 * **Dataset unique** - The field contains a value that maps to a single distinct entity within the column. For example, if a route is assigned the ID **1A**, then no other route may use that route ID. However, you may assign the ID **1A** to a location because locations are a different type of entity than routes.
 
+## Field Types
+
+- **Color** - The field contains a color encoded as a six-digit hexadecimal number. Refer to https://htmlcolorcodes.com/ to generate a valid value (the leading "#" is not included). For example: "FFFFFF" for white, "000000" for black or "0039A6" for the A,C,E lines in NYMTA.
+- **Currency Code** - The field contains an ISO 4217 alphabetical currency code. For the list of current currency, please refer to http://en.wikipedia.org/wiki/ISO_4217. For example: "CAD" for Canadian dollars, "EUR" for euros or "JPY" for Japanese yen.
+- **Email** - The field contains an email address. For example: example@example.com.
+- **Enum** - The field contains a value from a set of predefined constants defined in the "Details" column. For example: the route_type field contains a "0" for tram, a "1" for subway...
+- **ID** - The field contains an identifier which uniquely identifies an entity (therefore  is dataset unique). Its value is not aimed to be displayed to the user, and is a sequence of any UTF-8 characters, but using only printable ASCII characters is recommended.
+- **Language Code** - The field contains a IETF BCP 47 language code. For an introduction to IETF BCP 47, please refer to http://www.rfc-editor.org/rfc/bcp/bcp47.txt and http://www.w3.org/International/articles/language-tags/. For example: "en" for English, "en-US" for American English or "de" for German.
+- **Latitude** - The field contains the WGS84 latitude in decimal degrees. The value must be greater than or equal to -90.0 and less than or equal to 90.0. For example: "41.890169" for the Colloseum in Roma.
+- **Longitude** - The field contains the WGS84 longitude in decimal degrees. The value must be geater than or equal to -180.0 and less than or equal to 180.0. For example: "12.492269" for the Colloseum in Roma.
+- **Non-negative Float** - The field contains a non-negative floating point number (which includes 0).
+- **Non-negative Integer** - The field contains a non-negative whole number (which includes 0).
```

Comment:
If the previous change is accepted, then I suggest replacing the parens with a comma for consistency.

**Non-negative Integer** - The field contains a non-negative whole number, which includes 0.